components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIProvider.java

```diff
@@ -418,7 +418,7 @@ boolean updateAPIStatus(APIIdentifier apiId, String status,boolean publishToGate
      * @throws APIManagementException If an error occurs while trying to create
      *                                the new version of the API
      */
-    API createNewAPIVersion(String apiId, String newVersion, Boolean defaultVersion)
+    API createNewAPIVersion(String apiId, String newVersion, Boolean defaultVersion, String orgId)
```

Comment:
```suggestion
    API createNewAPIVersion(String apiId, String newVersion, Boolean defaultVersion, String organizationId)
```

Reply:
As we discussed offline, **orgId**  will be used in interfaces. On the implementation side, we use **tenantDomain** and **organizationId** .